README.md

```diff
@@ -35,7 +35,7 @@ This is quite slow and can take up to one or two hours but can provide good resu
 
 -   Go
 -   PHP
--   TypeScript
+-   [TypeScript and TSX](https://github.com/MaibornWolff/metric-gardener/blob/main/TS_TSX.md)
```

Comment:
Maybe we should continue to list them separately and link them to the same document? But I'm not sure, as both are quite similar.

Reply:
Also, I am not sure if it is that clear that we can click on the language name to find information about what we count for that language. We maybe should add somewhere in the readme a note that one can find language-specific details about the metrics in these files.